contracts/sdk/indexer/db.ts

```diff
@@ -278,6 +294,42 @@ export class NFTDatabaseConnection {
     return res;
   }
 
+  async getMissingDataWithTx(minSeq: number, treeId: string) {
```

Comment:
This probably doesn't need to be its own function. the right thing to do is to probably refactor `getMissingData` to return its own type. Punt for now

Reply:
Agreed